cypress/integration/examples/actions.spec.js

```diff
@@ -0,0 +1,299 @@
+/// <reference types="cypress" />
```

Comment:
In the future, I would recommend deleting these example files before committing and pushing your work. Makes it easier to review your code and ensures that you're only saving code that's relevant to the project.